test/Infer/demanded-bits4.opt

```diff
@@ -0,0 +1,11 @@
+; REQUIRES: solver, solver-model
+
+; RUN: %souper-check %solver -infer-rhs -souper-infer-inst -souper-synthesis-ignore-cost -souper-synthesis-comps=const,sub %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: %2:i64 = sub 3:i64, %0
+; CHECK-NEXT: result %2
+
+%0:i64 = var
+%1:i64 = subnuw 3:i64, %0
+infer %1 (demandedBits=0000000000000000000000000000000000000000000000000000000000000011)
```

Comment:
I don't understand the point of this test case, it works equally well when all bits are demanded

Reply:
Yes, ideally it should have all bits demanded. Obviously, it can never infer the exact value. I should make all bits demanded?

Reply:
No, you should make the test case make non-trivial use of demanded bits. this test case is not testing demanded bits in any interesting way.

Reply:
Sure, I will remove it and try to write some interesting test case.